spec/controllers/questions/backtaxes_controller_spec.rb

```diff
@@ -20,19 +20,36 @@
         }
       end
 
-      it "creates new intake backtaxes answers" do
-        expect {
-          post :update, params: params
-        }.to change(Intake, :count).by(1)
-
-        intake = Intake.last
-
-        expect(intake.source).to eq "source_from_session"
-        expect(intake.referrer).to eq "referrer_from_session"
-        expect(intake.locale).to eq "en"
-        expect(intake.needs_help_2017).to eq "no"
-        expect(intake.needs_help_2018).to eq "yes"
-        expect(intake.needs_help_2019).to eq "yes"
+      context "without an intake in the session" do
+        it "creates new intake backtaxes answers" do
+          expect {
+            post :update, params: params
+          }.to change(Intake, :count).by(1)
+
+          intake = Intake.last
+
+          expect(intake.source).to eq "source_from_session"
+          expect(intake.referrer).to eq "referrer_from_session"
+          expect(intake.locale).to eq "en"
+          expect(intake.needs_help_2017).to eq "no"
+          expect(intake.needs_help_2018).to eq "yes"
+          expect(intake.needs_help_2019).to eq "yes"
+        end
+      end
+
+      context "with an existing intake in the session" do
```

Comment:
@bengolder this is the test that checks that the backtaxes controller overwrites an existing intake (dimitri also linked to it i think)